packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2220,6 +2220,10 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// Removes all semantics from this render object and its descendants.
   ///
   /// Should only be called on objects whose [parent] is not a [RenderObject].
+  ///
+  /// Override this method if you instantiate new [SemanticsNode]s in an
+  /// overridden [assembleSemanticsNode] method to dispose those nodes.
```

Comment:
nit: comma before "to", otherwise you're saying that it should be overridden (but not saying why), if the person is disposing the nodes in their assembleSemanticsNode. :-)

Reply:
Thanks for including a grammatical explanation!